criteria/reusable-and-portable-codebases.md

```diff
@@ -7,6 +7,7 @@ order: 3
 ## Requirements
 
 * The codebase MUST be developed to be reusable in different contexts.
+* The codebase SHOULD be modularized, so that system parts can be replaced, separated and recombined and developed more independently.
```

Comment:
The two instances of the word "and" in the sentence invite a question about the grouping. For instance, does it mean

System parts can be:

*  replaced, separated and recombined
* and developed more independently.

or maybe system parts can be:

 * replaced
 * separated and recombined
 * developed independently


Can we make this sentence less ambiguous? Perhaps by reordering the elements, replacing the first and with a comma, adding an "oxford comma", or perhaps parenthesis?

Reply:
I would drop the word `system` in the sentence. I believe system isn't a wording used in the standard

I also miss the requirement for modules to have a certain generality or generic interfacing.